OmniBLE/OmnipodCommon/BasalDeliveryTable.swift

```diff
@@ -159,6 +159,24 @@ extension BasalDeliveryTable: CustomDebugStringConvertible {
     }
 }
 
+// Normalize basal rate by rounding down to pulse size boundary,
+// but basal rates within a small delta will be rounded up.
+func normalizedBasalRate(rate: Double) -> Double
```

Comment:
I think "rounding" is a better word for this process rather than "normalizing", and would prefer we keep this just a simple rounding that can be explained easily in those few cases we need to. 

Because this should only be used in certain edge cases that we're not currently handling in the UI yet, I don't think it justifies trying to be sophisticated about it. Standard rounding, or next lowest rate are fine.

Reply:
I saw values of 0.6499999999999 resulting from 0.65 from some test cases, so this is why I didn't couldn't just round down since this was breaking some test cases. Floating point can be a real pain. I really wish that we did everything in terms of pulses like the PDM does to avoid all these headaches. So then I wanted to avoid using rounding to try to avoid confusion with the similar funcs in the PumpManager files that I couldn't access. Will switch to use "round" instead of "normalize".